cfme/v2v/migrations.py

```diff
@@ -495,6 +532,33 @@ def sort_by(self, option):
             raise ItemNotFound("Migration plan is in Not Started State,"
                 " hence sort_by dropdown not visible")
 
+    def get_plan_status(self, vms_count=5):
+        """Reuturn True or False, depending on migration plan status.
```

Comment:
typo on 'Reuturn', and please format this docblock more - first line should describe the action/use for the caller.

These comments should be in `Returns:` header

Reply:
I think returning `True` on both any plan running, and no plans running, is an ambiguous implementation for the caller.

Why not return `None` when there are no plans in progress?

I think the function may be a bit mis-named, as its not actually returning a status, which I would expect from `get_plan_status`. Does True mean 'in progress'? finished? finished with success?

Consider either actually returning a status/`None` or renaming to `plan_in_progress` or something to that effect.

Reply:
^ agree